WalletWasabi.Backend/Controllers/SoftwareController.cs

```diff
@@ -11,18 +12,68 @@ namespace WalletWasabi.Backend.Controllers
 	[Route("api/[controller]")]
 	public class SoftwareController : Controller
 	{
-		private readonly VersionsResponse VersionsResponse = new VersionsResponse { ClientVersion = Constants.ClientVersion.ToString(3), BackendMajorVersion = Constants.BackendMajorVersion };
+		private readonly VersionsResponse VersionsResponse;
+
+		private const string LegalIssuesPath = "Assets/LegalIssues.txt";
+		private const string PrivacyPolicyPath = "Assets/PrivacyPolicy.txt";
+		private const string TermsAndConditionsPath = "Assets/TermsAndConditions.txt";
+
+		public SoftwareController()
+		{
+			VersionsResponse = new VersionsResponse()
+			{
+				ClientVersion = Constants.ClientVersion.ToString(3),
+				BackendMajorVersion = Constants.BackendMajorVersion,
+				LegalDocsVersion = Constants.LegalDocsVersion.ToString(4)
+			};
+		}
 
 		/// <summary>
-		/// Gets the latest versions of the client and backend.
+		/// Gets the latest versions of the client, backend and legal docs.
 		/// </summary>
-		/// <returns>ClientVersion, BackendMajorVersion.</returns>
-		/// <response code="200">ClientVersion, BackendMajorVersion.</response>
+		/// <returns>ClientVersion, BackendMajorVersion, LegalDocsVersion.</returns>
+		/// <response code="200">ClientVersion, BackendMajorVersion, LegalDocsVersion.</response>
 		[HttpGet("versions")]
 		[ProducesResponseType(typeof(VersionsResponse), 200)]
 		public VersionsResponse GetVersions()
 		{
 			return VersionsResponse;
 		}
+
+		/// <summary>
+		/// Gets the latest version of the LegalIssues document.
+		/// </summary>
+		/// <returns>LegalIssues text document bytes in BASE64 format.</returns>
+		/// <response code="200">LegalIssues</response>
+		[HttpGet("legalissues")]
+		[ProducesResponseType(typeof(byte[]), 200)]
+		public async Task<IActionResult> GetLegalIssuesAsync()
```

Comment:
I separated the downloadability of the three docs. It is more flexible this way. However, we only have one version string.